### ISSUE TITLE
Many queries predicting class 1

### ISSUE THREAD

**Issue by hvk280194** [PRBonn/MaskPLS#15]
Hello,

While using the pretrained model, I've observed that many queries in pred_logits are predicting class 1, which corresponds to cars. For each output I examined, a significant number of queries seem to be identifying class 1. Is this behavior due to the prevalence of multiple cars in a single frame of dataset, leading multiple queries to recognize different cars? 

Thanks

**Comment by hvk280194** [comment]
One more question,

Did you use any dropout while training?

Thanks
